client/hooks/useMuncher.js

```diff
@@ -0,0 +1,124 @@
+import { useState } from 'react';
+import newGame from '../GameState/muncherState.js';
+
+export function useMuncher() {
+  // I need to hand a new array to react
+  // will get moved to a hook eventually?
+  const [game, setGame] = useState(newGame);
+  const [targetSound, setTargetSound] = useState('ee');
+  const [gameover, setGameover] = useState(false);
+  const [matchesLeft, setMatchesLeft] = useState(4);
+  const [points, setPoints] = useState(0);
+  const [lives, setLives] = useState(['x', 'x', 'x']);
+  const [currentPosition, setCurrentPosition] = useState(0);
+
+  // get these targets from a fetch?
+  const targets = ['ee', 'ea', 'oo', 'ou'];
+
+  const checkMatches = () => {
+    // when resetting the phonics fetch a whole new game for those sounds
+    if (matchesLeft < 1) {
+      if (targetSound === 'ee') {
+        setTargetSound('ea');
+        setMatchesLeft(4);
+      }
+      else {
+        setTargetSound('ee');
+        setMatchesLeft(4);
+      }
+    }
+  };
+
+  const calculatePoints = (sign) => {
+    if (sign === '+') {
+      setPoints(prev => prev + 200);
+    }
+    else {
+      setPoints(prev => prev - 200);
+    }
+  };
+
```

Comment:
This might be a moment for a ternary operation to avoid the two separate function calls:
```js
setPoints(prev  => sign === '+' ? prev + 200 : prev - 200)

Reply:
that is a good point, will do that